### DISCUSSION TITLE
is MAQS-dotnet still being maintained?

### DISCUSSION THREAD

**Discussion by pjotal** [CognizantOpenSource/maqs-dotnet#306] [General]
hi,

the latest release of maqs-dotnet is from 2022 so i am wondering if cognizant is still maintaining this framework or not.
maybe the framework is now under a new name? if so, could you please reply with the link?

there are several dependencies that needs to be bumped up in maqs-dotnet, including selenium webdriver.

found this but i am not sure if it's developed and maintained by cognizant or not: https://github.com/OpenMAQS/openmaqs-dotnet

let us know.
thanks.

**Comment by bobbyislamtoro** [comment]
I would also like to know the roadmap for MAQS. We are using v9.0.3 and would like to keep utilizing MAQS for our automation.
